src/content/docs/en/reference/api-reference.mdx

```diff
@@ -2495,15 +2495,15 @@ Actions help you build a type-safe backend you can call from client code and HTM
 <Since v="4.15.0" />
 </p>
 
-The `defineAction()` utility is used to define new actions from the `src/actions/index.ts` file. This accepts a `handler()` function containing the server logic to run, and an optional `input` property to validate input parameters at runtime.
+The `defineAction()` utility is used to define new actions from the `src/actions/index.ts` file. This accepts a [`handler()`](#handler-property) function containing the server logic to run, and an optional [`input`](#input-validator) property to validate input parameters at runtime.
 
 ```ts
 export const server = {
-    getGreeting: defineAction({
+  getGreeting: defineAction({
     input: z.object({
       name: z.string(),
     }),
-    handler: async (input) => {
+    handler: async (input, context) => {
       return `Hello, ${input.name}!`
     }
```

Comment:
I think it's a good idea to show that context is also available, but shouldn't we update the example to include context?

This is a rough draft (I haven't taken the time to familiarize myself with Actions yet to find a better example) but maybe something like:
```suggestion
    handler: async (input, context) => {
      if (!input.name) context.redirect();
      return `Hello, ${input.name}!`
    }
```

Reply:
`context.redirect()` is not available on this version of context 😅 

I also thought about this, but I mainly wanted to show it exists here without muddying the example. I think the reason `context` was excluded in the initial version is that it’s mostly for pretty advanced use cases (for example it allows you to read `context.locals` if you have some middleware), so I think it’s OK not to demonstrate it — just to mention it’s existence.

Reply:
Ah, I did say that I wasn't familiar enough with this feature. 😅 It's good for me so, no need for an overly complex example!